### PR TITLE
Refactor code 

### LINKED ISSUE #5: Refactor code
Pep8, refactor code into more modularized version, etc.

### PR DESCRIPTION
**Pull request by Nikoleta-v3**
Closes #5, #15 

Commit 595968e moves all the files under `src` folder. A setup.py file has been added which allow the use the package after running python setup.py

The `setup.py` files also create all folders needed if they do not exist as discussed in #15.

I also updated the `README` file to include installation and usage notes (commit 48040e6). 
I did however delete your original message. I am happy to undo the change :smile:

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,8 @@
 
-
 *.pyc
 
+# data
 data/builds/*
```
Comment (LTKills) [inline .gitignore]: if we are ignoring the data directory we shouldn't need to ignore anything under it (this line should be removed)

File: README.md [hunk 2]
```diff
@@ -1,16 +1,31 @@
...
 
-I've played Heroes of The Storm since its alpha releases but for some reason I've always found myself
-digging builds in Icy Veins instead of comming up with my own. It was all flowers when I had two
-monitors, but now that I'm back to single monitor I have to keep alt + tabbing whenever I have to
-get a talent. Hopefully this project will help me by going to the page and telling me which talent
-I need to get without leaving the game.
-
-## Dependencies
-For now the only dependency needed for this project is `bs4`
+`heroes-build-scrapper` is a packaged designed to scrape character builds for
```
Comment (LTKills) [inline README.md]: Typo: package* not packaged

File: README.md [hunk 3]
```diff
@@ -1,16 +1,31 @@
...
+the [moda](https://en.wikipedia.org/wiki/Multiplayer_online_battle_arena)
```
Comment (LTKills) [inline README.md]: Typo: MOBA*, not MODA. Also put it in caps

File: README.md [hunk 4]
```diff
@@ -1,16 +1,31 @@
...
 ```
-pip install bs4
+$ git clone https://git@github.com:LTKills/heroes-build-scrapper.git
+$ cd heroes-build-scrapper
+$ python setup.py install
 ```
+
 ## Usage
 
+Once you have install the package you are ready to collect some builds. This is
```
Comment (LTKills) [inline README.md]: Typo: installed*, not install

File: README.md [hunk 5]
```diff
@@ -1,16 +1,31 @@
...
+done using the following code::
```
Comment (LTKills) [inline README.md]: Typo: two colons (`:`)

File: setup.py [hunk 6]
```diff
@@ -0,0 +1,19 @@
+import os
+from setuptools import setup, find_packages
+
+folder = 'data/builds'
+if not os.path.exists(folder):
+    os.makedirs(folder)
+
+with open('requirements.txt') as f:
```
Comment (LTKills) [inline setup.py]: :bike: (bike means optional)

how about `fp` (file pointer) instead of `f`?

File: src/heroes_build_scrapper/data.py [hunk 7]
```diff
@@ -73,12 +75,6 @@ def load_builds(hero):
     with open(filename, 'r') as fp:
         load_list = json.load(fp)
 
-    for i in range(0, len(load_list) - 1, 2):
```
Comment (LTKills) [inline src/heroes_build_scrapper/data.py]: I'm sorry but I didn't understand what is going on here, what does the function returns then?

**Comment by LTKills** [comment]
Also, it'd be nice to have an `__init__.py` as explained [here](https://packaging.python.org/tutorials/packaging-projects/#a-simple-project)

File: .gitignore (hunk 1 above)
Comment (Nikoleta-v3) [reply .gitignore]: `# data` it's just an inline comment. I am not ignoring the data folder.

File: src/heroes_build_scrapper/data.py (hunk 7 above)
Comment (Nikoleta-v3) [reply src/heroes_build_scrapper/data.py]: Good catch!!! That's a mistake.

I rebased after your `pr` #21. That's when the mistake occurred.

**Comment by Nikoleta-v3** [comment]
Do you mean this file: https://github.com/Nikoleta-v3/heroes-build-scrapper/blob/refactor-code/src/heroes_build_scrapper/__init__.py?

**Comment by Nikoleta-v3** [comment]
p.s. remember to change your email in `setup.py`

File: .gitignore (hunk 1 above)
Comment (Nikoleta-v3) [reply .gitignore]: No worries at all! Will do some work on this later today :smile:

**Pull request merged**